Assets/MixedRealityToolkit/Utilities/Editor/FileUtilities.cs

```diff
@@ -42,7 +42,7 @@ public static FileInfo[] FindFilesInPackageCache(string fileName)
         /// </returns>
         public static DirectoryInfo GetPackageCache()
         {
-            const string packageCacheFolderName = @"Library\PackageCache";
```

Comment:
I feel like we do this `@"Library\PackageCache";` kind of paths in a lot of MRTK. Might be worth opening an issue to investigate?

Reply:
#7132